pkgs/data/icons/vimix-cursor-theme/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "vimix-cursor-theme";
+  version = "2020-02-24";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = "Vimix-cursors";
+    rev = finalAttrs.version;
+    sha256 = "sha256-TfcDer85+UOtDMJVZJQr81dDy4ekjYgEvH1RE1IHMi4=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/icons/Vimix{,-white}-cursors
+    cp -r dist/{cursors,index.theme} $out/share/icons/Vimix-cursors
+    cp -r dist-white/{cursors,index.theme} $out/share/icons/Vimix-white-cursors
+    runHook postInstall
```

```suggestion
    runHook preInstall

    mkdir -p $out/share/icons/Vimix{,-white}-cursors
    cp -r dist/{cursors,index.theme} $out/share/icons/Vimix-cursors
    cp -r dist-white/{cursors,index.theme} $out/share/icons/Vimix-white-cursors

    runHook postInstall
```
just a small  formating nitpick